API-Samples/Java/app/src/main/java/videoindexersamples/HttpUtils/Utils.java

```diff
@@ -0,0 +1,50 @@
+package videoindexersamples.HttpUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLEncoder;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+public class Utils {
+    public static final int HTTP_OK = 200;

```

Comment:
Optional: Use https://hc.apache.org/httpclient-legacy/apidocs/org/apache/commons/httpclient/HttpStatus.html

Reply:
thought about it , it requires to bring Apache.Common dependecy.